### PR TITLE
Add conda-forge badge

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Add conda-forge badge and instructions to install kedro using `conda-forge` channel

## Checklist

- [ ] Read the [contributing](../CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change and added my name to the list of supporting contributions in the [`RELEASE.md`](../RELEASE.md) file
- [ ] Added tests to cover my changes

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 |------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Latest Release | [![PyPI version](https://badge.fury.io/py/kedro.svg)](https://pypi.org/project/kedro/) |
 | Python Version | [![Python Version](https://img.shields.io/badge/python-3.6%20%7C%203.7%20%7C%203.8-blue.svg)](https://pypi.org/project/kedro/) |
+| Conda-forge version | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/kedro.svg)](https://anaconda.org/conda-forge/kedro) |
```
Comment (mzjp2) [inline README.md:9]:
```suggestion
| Latest Release | [![PyPI version](https://badge.fury.io/py/kedro.svg)](https://pypi.org/project/kedro/) [![Conda Version](https://img.shields.io/conda/vn/conda-forge/kedro.svg)](https://anaconda.org/conda-forge/kedro) |
| Python Version | [![Python Version](https://img.shields.io/badge/python-3.6%20%7C%203.7%20%7C%203.8-blue.svg)](https://pypi.org/project/kedro/) |
```

What do you think about this instead? Happy either way though!

File: README.md [hunk 2]
```diff
@@ -34,6 +35,14 @@ pip install kedro
 
 See more detailed installation instructions, including how to setup Python virtual environments, in our [installation guide](https://kedro.readthedocs.io/en/stable/02_getting_started/02_install.html) and get started with our ["Hello Word"](https://kedro.readthedocs.io/en/stable/02_getting_started/04_hello_world.html) example.
 
+### Conda package
+
+It is also possible to install `kedro` using `Conda` with the `conda-forge` channel. With `conda` already installed, simply run:
```
Comment (mzjp2) [inline README.md:40]:
```suggestion
It is also possible to install `kedro` using Conda with the `conda-forge` channel. With `conda` already installed, simply run:
```

maybe? :)

File: README.md (hunk 1 above)
Comment (marcelotrevisani) [reply README.md:9]: That is fine for me :)
If you want to change it please fell free for it, because I think it was already merged :grin:

Comment (yetudada) [reply README.md:9]: We'll take care of it 😄 Thank you!

**Pull request merged**